### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by abundis28**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: Template
+about: 'Standardize issues. '
+title: ''
```
Comment (shaargtz) [inline .github/ISSUE_TEMPLATE/template.md:4]: What is the difference between name and title?

**Comment by shaargtz** [comment]
Btw remember to name the branch in this case docs/issue-template

File: .github/ISSUE_TEMPLATE/template.md (hunk 1 above)
Comment (abundis28) [reply .github/ISSUE_TEMPLATE/template.md:4]: I believe title is the specific title for each issue, whilst name is just for the template itself.

**Comment by abundis28** [comment]
I didn't notice that a folder was created, I certainly did not create intentionally so I guessed GitHub created it itself.

**Pull request merged**